measure.rb

```diff
@@ -3388,7 +3398,7 @@ def self.create_ducts(air_distribution, model, spaces)
   end
 
   def self.add_hvac_sizing(runner, model, weather)
-    success = HVACSizing.apply(model, runner, weather, @cfa, @infilvolume, @nbeds, @min_neighbor_distance, false, @living_space)
+    success = HVACSizing.apply(model, runner, weather, @cfa, @infilvolume, @nbeds, @min_neighbor_distance, false, @living_space, @living_zone)
```

Comment:
This doesn't really need to be passed, since you can always do `@living_space.thermalZone.get` inside HVACSizing.apply().